### PR TITLE
Fix: Include .library and manifest files as part of packaging 

### PR DESCRIPTION
**Pull request by AmirthavalliG**
Npm packaging doesn't contain the .library.
Which is needed by the ui5 builder when added as dependency .  
This pr ensures that those files are added by default. 
Also change the spec version

### CODE REVIEW

**Comment by AmirthavalliG** [comment]
https://github.com/SAP/ui5-tooling/issues/529

File: scripts/build/replaceCopyright.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const yamlPath = "ui5.yaml";
 
 function getDefaultCopyRight() {
     const content = replaceCurrentYear(fs.readFileSync(yamlPath, 'utf8'));
-    const yamlContent = yaml.safeLoad(content);
+    const yamlContent = yaml.load(content);
```
Comment (flovogt) [inline scripts/build/replaceCopyright.js:8]: does the safeLoad no longer exists?
Does this belongs to this fix? If not I would create another PR fixing the load function first and then merge this PR

**Review by flovogt — CHANGES_REQUESTED** [review]

File: scripts/build/replaceCopyright.js (hunk 1 above)
Comment (AmirthavalliG) [reply scripts/build/replaceCopyright.js:8]: @flovogt : i saw the error when i tried to run in local. anyways raised another pr #281

**Review by flovogt — APPROVED** [review]
LGTM 🎉

**Pull request merged**